sema4ai/vscode-client/src/robo/actionPackage.ts

```diff
@@ -659,3 +659,33 @@ export const publishActionPackage = async () => {
         }
     );
 };
+
+export const buildActionPackage = async() => {
+    const workspaceDir = await askForWs();
+
+    await window.withProgress(
+        {
+            location: ProgressLocation.Notification,
+            title: "Build Action Package",
+            cancellable: false,
+        },
+        async (
+            progress: Progress<{ message?: string; increment?: number }>,
+            token: CancellationToken
+        ): Promise<void> => {
+            progress.report({ message: "Validating action server" });
+            const actionServerLocation = await downloadOrGetActionServerLocation();
+            if (!actionServerLocation) {
+                return;
```

Comment:
We need some error handling here.
A message to the user would suffice, but probably this will fail more than it will go through.

Reply:
Same as above.